x/bank/keeper/keeper.go

```diff
@@ -116,85 +116,89 @@ func (k BaseKeeper) WithMintCoinsRestriction(check types.MintingRestrictionFn) B
 	return k
 }
 
+// NOTE(Heimdall-v2): vesting accounts are not used in heimdall
```

Comment:
Sure, it’s not about winning the argument 🙏
iirc we had this convo on internal slack channel
If you want to keep the “Note” one, just please let me know so that I can update my auth and gov PR 
If that’s the case, please post a msg in slack with the updated comments for both TODOs e and Note so that everyone uses them

Reply:
Btw if we change this in `auth`, all PRs raised against auth branch must pull the target, just to avoid conflicts afterwards

Reply:
Well was just giving 2c 😢 
I don't recall having a convo on this, my bad. I'll update the comment format since it was already agreed.

Reply:
Done [629bdd5](https://github.com/0xPolygon/cosmos-sdk/pull/5/commits/629bdd5911ec2c9ad403991b07d3d3f21db7b4ae).